### DISCUSSION TITLE
what about py5 logo?

### DISCUSSION THREAD

**Discussion by Maltiec** [py5coding/py5generator#284] [General]
This may be untimely, but I looked at the back of my laptop today and saw a sticker with classic Processing logo (I donated money to Processing foundation in 2019, and in 2020 found some great stickers in my mailbox). I think that it would be great to have a sticker with py5 logo on my laptop. ) @hx2A may be you already made py5 logo (I saw a nice picture at py5coding website), or may be thinking about it?

**Comment by hx2A** [comment]
Hmmm, what a nice idea! @Maltiec , thanks for asking this question. I, too, have looked at the stickers for other open source projects and thought about what it would be like to see a py5 sticker as well. I am not really sure how to go about this. Where do these stickers come from? Are there certain companies everybody is using? I wouldn't mind having a py5 mug and t-shirt as well. Is there a company that can handle all of those things?

Right now I am focused on getting the next release out the door. Once that is done, I'll return to this and make something happen.

**Reply by hx2A** [reply]
Hmmm, it seems I can setup something on https://www.teepublic.com/ or https://www.cafepress.com/ and they will take care of it for me. Do either of those work for international sales? py5 has a large international group of users so I need to be sure what I do works for everyone.

**Reply by Maltiec** [reply]
I'll check this services if they work with customers from Russia (though I doubt, almost everyone stopped to ship here, because of war).

**Reply by Maltiec** [reply]
cafepress has limited number of countries for shipping, I can make an order though through my friends in Germany. I suppose we need to make a list of countries where members of py5 society live.

**Reply by hx2A** [reply]
So, cafepress would work for you through friends in Germany, but not teepublic? I can create a high-res image of the py5 logo and upload it to cafepress.

I can look at the website logs to see where site visitors are coming from. In March, the top countries were USA, Russia, Canada, Germany, Brazil, France, Austria, Great Britian, Ireland, and Poland. The full list is extensive, with visitors from every continent.

**Reply by Maltiec** [reply]
teepublic probably too, I'll ask them

**Comment by villares** [comment]
> This may be untimely, but I looked at the back of my laptop today and saw a sticker with classic Processing logo (I donated money to Processing foundation in 2019, and in 2020 found some great stickers in my mailbox). I think that it would be great to have a sticker with py5 logo on my laptop. ) @hx2A may be you already made py5 logo (I saw a nice picture at py5coding website), or may be thinking about it?

I have sent you a message about this at the discourse.processing.org forum's personal inbox... the thing is we'd like to send you some stickers. You can also reach me by email `abav` (at) `lugaralgum.com` :)
